coverage.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -ex
+
+# Merge coverage reports from cypress and jest tests
+mkdir -p reports
+cp cypress-coverage/coverage-final.json reports/from-cypress.json
```

Comment:
It's not included because it's absent. running coverage.sh will result in an error
cp: cannot stat 'jest-coverage/coverage-final.json': No such file or directory